### ISSUE TITLE
Model Card

### ISSUE THREAD

**Issue by creative-mjsapps** [Mjs-Apps/unmute#2]
# Unmute – Model Card

## 📌 Model Details
- **Model name**: Unmute
- **Base model**: (e.g., `meta-llama/Llama-2-7b` or other OSS model)  
- **Version**: v0.1
- **Developed by**: `<your name / org>`
- **Date released**: `<YYYY-MM-DD>`
- **License**: Apache-2.0 (repo), base model license may differ

Unmute is a lightweight open-source evaluation harness + model declaration for reproducible testing of GPT-style language models.  
It provides:
- Inference runner (`src/inference.py`)
- CI tests (`pytest`)
- Sample prompts (`sample_data/prompts.jsonl`)
- Evaluation rubric (see below)

---

## 🎯 Intended Use
- **Primary use case**: Research, evaluation, and reproducible testing of open-source GPT models.  
- **Not for**: Production deployment in sensitive domains (medical, legal, safety-critical tasks) without further alignment, testing, and domain review.

---

## ⚙️ Training Data & Method
- **Base model**: (fill in with exact base weights, e.g., *Llama-2-7b, MPT-7B, Falcon-7B, etc.*)
- **Fine-tuning**: 
  - [ ] None (vanilla base model)  
  - [ ] Yes — provide link to dataset / config in `TRAINING_LOG.md`
- **Data license**: (cite if fine-tuned)
- **Tokenizer**: same as base model

---

## 📊 Evaluation
Unmute includes:
- **Automated checks** (`pytest`):
  - Model responds to prompts without crashing
  - Outputs non-empty text
- **Human evaluation** (`sample_data/prompts.jsonl` + `human_eval.md`):
  - *Fluency*: Is the text grammatical, coherent, natural?  
  - *Relevance*: Does it answer the prompt?  
  - *Factuality*: Does it contain hallucinations?  
  - *Toxicity*: Any unsafe or biased language?  

### Example Results (fill once you test):
| Metric        | Result |
|---------------|--------|
| Fluency       | ✅ High |
| Relevance     | ✅ Strong |
| Factuality    | ⚠️ Mixed |
| Toxicity risk | Low |

---

## ⚠️ Limitations
- **Hallucinations**: May generate factually incorrect info.
- **Biases**: Can reflect stereotypes present in training data.
- **Compute requirements**: Running large models (7B+) requires a GPU with >16GB VRAM, otherwise use quantized versions.
- **Context length**: Limited by the base model (e.g., 4k tokens).

---

## 🔒 Ethical Considerations
- Do not use Unmute outputs in domains requiring professional expertise (e.g., medicine, law).
- Users must verify outputs independently before use.
- Be cautious with prompts that could elicit unsafe or harmful content.

---

## 📦 How to Reproduce
1. Clone repo:
   ```bash
   git clone https://github.com/<your-org>/unmute
   cd unmute
